src/bin/psql/describe.c

```diff
@@ -1959,7 +1961,8 @@ describeOneTableDetails(const char *schemaname,
 						  "false AS relhasoids, c.relispartition, %s, c.reltablespace, "
 						  "CASE WHEN c.reloftype = 0 THEN '' ELSE c.reloftype::pg_catalog.regtype::pg_catalog.text END, "
 						  "c.relpersistence, c.relreplident, am.amname, "
-						  "c.relisivm\n"
+						  "c.relisivm, "
```

Comment:
Also need to support pg_dump .

Reply:
> And `tab-complete.c` suport this syntax hint.

See https://github.com/apache/cloudberry/pull/725#issuecomment-2490804387, Tab is optional.

Reply:
> Also need to support pg_dump .

Yes, good catch.

Reply:
Fixed.